pkg/virt-controller/services/template.go

```diff
@@ -259,12 +260,27 @@ func (t *templateService) GetLauncherImage() string {
 }
 
 func (t *templateService) RenderLaunchManifestNoVm(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, error) {
-	return t.renderLaunchManifest(vmi, nil, true)
+	backendStoragePVCName := ""
+	if backendstorage.IsBackendStorageNeededForVMI(&vmi.Spec) {
+		backendStoragePVC := backendstorage.PVCForVMI(t.persistentVolumeClaimStore, vmi)
+		backendStoragePVCName = backendStoragePVC.Name
+		if backendStoragePVCName == "" {
+			return nil, fmt.Errorf("can't generate manifest without backend-storage PVC")
```

Comment:
nit: can you change it into `"can't generate manifest without backend-storage PVC, waiting for the PVC to be created"` or something similar?

I think it's better to convey the fact that this could be a transient error (we're just waiting for the PVC to appear).